src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableValueStore.kt

```diff
@@ -795,4 +816,24 @@ class VariableValueStore(
     return dslContext.fetchValue(VARIABLE_VALUES.PROJECT_ID, VARIABLE_VALUES.ID.eq(valueId))
         ?: throw VariableValueNotFoundException(valueId)
   }
+
+  private fun notifyForReview(
+      projectId: ProjectId,
+      values: List<ExistingValue>,
+  ) {
+    val valuesByDeliverables =
+        values.groupBy {
+          val variablesRow =
+              variablesDao.fetchOneById(it.variableId)
+                  ?: throw VariableNotFoundException(it.variableId)
+          variablesRow.deliverableId
+        }
+
+    valuesByDeliverables.keys.forEach { deliverableId ->
+      val thisValues = valuesByDeliverables[deliverableId]!!.associate { it.variableId to it.id }
+      deliverableId?.let {
```

Comment:
Not including null deliverable IDs in `valuesByDeliverables` is another option; that way it can also skip building `thisValues`.